core/domain/exp_jobs_one_off.py

```diff
@@ -252,6 +252,60 @@ def reduce(key, values):
         yield (key, values)
 
 
+class ExplorationMigrationAuditJob(jobs.BaseMapReduceOneOffJobManager):
+    """A reusbale one-off job for testing exploration migration from the
+    previous exploration schema version to the latest. This job runs the state
+    migration, but does not commit the new exploration to the store.
```

Comment:
store --> datastore. (OK to fix in a future PR)

Reply:
Done